pkgs/by-name/fi/firefoxpwa/package.nix

```diff
@@ -1,22 +1,28 @@
-{ lib
-, rustPlatform
-, fetchFromGitHub
-, makeWrapper
-, pkg-config
-, installShellFiles
-, firefox-unwrapped
-, openssl
-, stdenv
-, udev
-, libva
-, mesa
-, libnotify
-, xorg
-, cups
-, pciutils
-, libcanberra-gtk3
-, extraLibs ? [ ]
-, nixosTests
+{
```

Comment:
Do not modify its order. This order is based on the order they are called below.